server/server.go

```diff
@@ -397,13 +397,15 @@ func createQueryResolver(
 	redisClient *redis.Client,
 ) (r resolver.Resolver, err error) {
 	blocking, blErr := resolver.NewBlockingResolver(cfg.Blocking, redisClient, bootstrap)
+	rErr := cfg.Upstream.AddResolvconfToConfig()
```

Comment:
I don't see the implementation for this function anywhere. Did you forget to stage it?

Reply:
Upsi